source/NVDAObjects/IAccessible/__init__.py

```diff
@@ -1313,7 +1313,7 @@ def _getSelectedItemsCount_accSelection(self, maxCount: int) -> int:
 			raise NotImplementedError
 		# accSelection can return a child ID of a simple element, for instance in QT tree tables. 
 		# Therefore treat this as a single selection.
-		if isinstance(sel,int) and sel>0:
+		if isinstance(sel, int):
 			return 1

```

Comment:
Shouldn't this still compensate for 0? I don't think we should treat CHILDID_SELF as a selection within the container.

```suggestion
			return 0 if sel == 0 else 1
```

Reply:
You're right, thanks!